Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Blobs"
 uuid = "163b9779-6631-5f90-a265-3de947924de8"
 authors = []
-version = "0.5.1"
+version = "1.1.0"
```

Comment:
not sure what's happening here. Why is this a breaking change?

Reply:
🤔 Doesn't seem like one? 👍

Reply:
Oh, i see, this is what happened:
https://github.com/RelationalAI-oss/Blobs.jl/pull/30.

I wanted to release a 1.0, so that this could be a minor release (which is what it should be, since only bugfixes should be patch releases).